Week01/Problem04/MZand_04.py

```diff
@@ -0,0 +1,42 @@
+
+import math
+
+
+def Palindromic(x):
```

Comment:
Kudos to you, this bit of code is really strange...how'd you come up with reversing the number this way?  I would've just cast it to a string, then reversed it:
```
def str_palindrome(val):
    strval = str(val)
    if strval == strval[::-1]:
        return True
```